fluent-resmgr/src/resource_manager.rs

```diff
@@ -48,21 +48,25 @@ impl ResourceManager {
 
     /// Returns a [`FluentResource`], by either reading the file and loading it into
     /// memory, or retrieving it from an in-memory cache.
-    fn get_resource(&self, resource_id: &str, locale: &str) -> &FluentResource {
+    fn get_resource(
+        &self,
+        res_id: &str,
+        locale: &str,
+    ) -> Result<&FluentResource, ResourceManagerError> {
         let path = self
             .path_scheme
             .replace("{locale}", locale)
-            .replace("{res_id}", resource_id);
-        if let Some(res) = self.resources.get(&path) {
+            .replace("{res_id}", res_id);
```

Comment:
Nit: I'd prefer to have the full name written out, as `res` can be a little mysterious. I know previous authors used smaller names, but I find being explicit makes the code so much easier to read and understand when you have low context for changes.